paderbox/utils/random_utils.py

```diff
@@ -177,6 +179,16 @@ def _sample(self, shape):
         return truncexpon(self.truncation / self.scale, self.loc, self.scale).rvs(shape)
 
 
+@dataclasses.dataclass
+class Choice(_Sampler):
+    events: int = 1
```

Comment:
`Choice` and `choice` should have the same default for `events`.
While other distributions have some kind of natural defaults, I would say, we shouldn't set a default for events.